openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    transformation = openfe.Transformation(
</pre><p> </p><p>could we use a <code>NonTransformation</code> gufe class here?</p><p>That way we could get rid of SystemB :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
https://github.com/OpenFreeEnergy/gufe/blob/45273645f99aaea1ac748854496f076bbabbfeab/gufe/transformations/transformation.py#L243-L259

Reply:
<p>Thank you, I changed this!</p>